node/src/actors/messages.rs

```diff
@@ -959,3 +959,14 @@ pub struct SuperBlockNotify {
     /// The hashes of the blocks that we are signaling as consolidated.
     pub consolidated_block_hashes: Vec<Hash>,
 }
+
+/// Notification signaling that the node's state has changed.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct ChangeNodeStatusNotify {
+    /// The node status.
+    pub node_status: StateMachine,
```

Comment:
I think a client only needs to know if the node is synced or not, so what do you think about changing this into a simple flag? This way we would avoid moving `StateMachine` to `chain.rs`

```suggestion
    pub node_synced: bool,
```

Reply:
I kind of like more serving raw states as they give clients the chance to offer more insightful and helpful information about what's up with the node they are using in the back-end.